CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 5.0.0 (2022-07-15)
```

Comment:
This can be 14th of July 2022

Reply:
I left that as placeholder, since I wasn't sure. We can merge today lol